### DISCUSSION TITLE
Raise Event Filter Change

### DISCUSSION THREAD

**Discussion by carrisian** [dotnet/DataGridExtensions#79] [Q&A]
Is it possible to raise an event every time something in the filters is modified? Is there an example of how to do that?

Thank you in advance

**Comment by tom-englert** [comment]
https://github.com/dotnet/DataGridExtensions/blob/4c983838264b5699c0d307a617767a1a21ef51b1/src/DataGridExtensions/DataGridFilterHost.cs#L66-L69
